solutions/devsprint-michael-douglas-2/DeliveryApp/Screens/Home/HomeView.swift

```diff
@@ -26,6 +26,14 @@ final class HomeView: UIView {
         tableView.dataSource = self
         return tableView
     }()
+    
+    private var loadingView: LoadingView = {
+        let loadingView = LoadingView()
+        loadingView.translatesAutoresizingMaskIntoConstraints = false
+        loadingView.loadingSpinner.startAnimating()
```

Comment:
Considere adicionar o `startAnimating()` dentro do seu método de `updateLoading` para ser chamado quando for `true`, e quando for `false` lembre-se de chamar o `stopAnimating()`.

Reply:
Uma outra coisa, é criar esse método de `updateLoading` dentro da própria classe de `LoadingView`.